### ISSUE TITLE
NerdMiner runs out of KH/s

### ISSUE THREAD

**Issue by ricardonegrao** [BitMaker-hub/NerdMiner_v2#331]
This problem is occurring and I have already reset the equipment several times.

Has anyone ever experienced this?

Is there a solution?

![IMG_3111](https://github.com/BitMaker-hub/NerdMiner_v2/assets/40274281/2e6071a3-fa3a-4e45-a80a-b21d7f2952b9)

**Comment by 19201003080114** [comment]
Either wrong settings, or you try to connect to 5G wifi. 
Nerdminer only works on 2.4G WiFi.

**Comment by ricardonegrao** [comment]
I am using 2G.

How would you know what is happening with this equipment?

**Comment by 19201003080114** [comment]
Rest it and check your pool and port settings

**Comment by ricardonegrao** [comment]
This is my pool configuration:

Pool url pool.vkbit.com
Pool port 3333
-3

**Comment by 19201003080114** [comment]
Try resetting your wifi router.

**Comment by DanielErnany** [comment]
> I found the problem:
> 
> It was my BTC wallet address!
> 
> Thank you for your help.

What was wrong with her? and how did you fix it? I have the same problem

**Comment by v1per4llbl4ck** [comment]
me the same i dont know how to change btc address. with sdcard dont work. i take my address from crypto.com in receve mode

**Comment by 19201003080114** [comment]
> me the same i dont know how to change btc address. with sdcard dont work. i take my address from crypto.com in receve mode

Boot to Wifi Manger and you can change the btc address

**Comment by 19201003080114** [comment]
> > I found the problem:
> > It was my BTC wallet address!
> > Thank you for your help.
> 
> What was wrong with her? and how did you fix it? I have the same problem

Boot to Wifi Manger and you can change the btc address

**Comment by MARK345456** [comment]
ALGUEM SABE COMO REINICIAR O NERDMINER

**Comment by dwightmulcahy** [comment]
If you see your Nerdminer go to 0.00kH/s a lot, it could be the cable or power source you are using.  I noticed that it bounces between 78kH/s and 0.00kH/s when I have it hooked up to my laptop, but when I plug it into a 1 amp or better power supply, it stays at the 78kH/s-ish mining all the time.

**Comment by ricardonegrao** [comment]
https://flasher.bitronics.store/

Em qui., 7 de mar. de 2024 às 16:20, MARK345456 ***@***.***>
escreveu:

**Comment by ben6608** [comment]
I also have the same problem. I updated the BTC wallet address and it didn't improve. I also changed the pool and reset the flash machine, but it didn't work.

**Comment by dwightmulcahy** [comment]
Have you tried a different power supply and cable to eliminate that?

**Comment by ben6608** [comment]
Yes, I tried changing the power supply and cables, but it didn’t work.

**Comment by 19201003080114** [comment]
> Yes, I tried changing the power supply and cables, but it didn’t work.

Check your settings:
- It can only connect to 2.4 Ghz wifi
- Make sure SSID and password are correct
- Make sure pool address, pool port and BTC wallet address are correct

**Comment by Loickrypto** [comment]
same issue . the problem was the btc adress . works well now

**Comment by Blackhawk17-USA** [comment]
1. I use Public Pool so this is my Pool Config stratum+tcp://public-pool.io:21496
2. My BTC Address has always been good
3. I'm connected to the correct WiFi at 2.4 Ghz
4. I successfully flashed NerdMinerv2 to 1.6.3 original board T-Display S3
5. I removed USB cable from computer and plugged into 5 Watt USB adapter  👍👍👍 
Seems like the only thing that fixed it, was number 5, the POWER ADAPTER upgrade. LOL
Thank you Dwight Mulcahy 💯

**Comment by andrewlimaza** [comment]
My WIFI SSID had spaces in the name. I created a new guest network with no spaces "guestnetwork" and now it connects 100% of the time. Hope this helps anyone.

**Comment by brunoj893-coder** [comment]
I'm having the same problem, what exactually was wrong with your btc adress?

**Comment by raysomerville2023-creator** [comment]
Use the nerdminer.org address 
stratum+tcp://pool.nerdminers.org:3333

**Comment by brunoj893-coder** [comment]
just tried that, the error persist...  should i try to create a new wallet and try to use it to acess?
